packages/syntax-core/src/RadioButton/RadioButton.module.css

```diff
@@ -0,0 +1,82 @@
+.md {
+  height: 24px;
+  width: 24px;
+  border-radius: 12px;
+}
+
+.sm {
+  height: 16px;
+  width: 16px;
+  border-radius: 8px;
+}
+
+.smBase {
+  height: 16px;
+}
+
+.mdBase {
+  height: 24px;
+  gap: 8px;
+}
+
+.baseRadioButton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  cursor: pointer;
+}
+
+.labelTypography {
+  font-style: normal;
+  font-weight: 510;
+  font-size: 14px;
+  line-height: 17px;
+}
+
+.radioStyleOverride {
+  opacity: 0;
+}
+
+.background {
+  position: absolute;
+  box-sizing: border-box;
+  border: 2px solid;
+  border-radius: 12px;
+}
+
+.errorBorderColor {
+  border-color: var(--color-base-destructive-700);
+}
+
+.borderColor {
+  border-color: var(--color-base-primary-700);
+}
+
+.mdInner {
+  background-color: #fff;
+  border-radius: 5px;
```

Comment:
Can we just use 50% for this border radius and the radius on L64? That way if height/width ever change, you don't need to change this again

Reply:
you might actually just be able to make a circle div base class that has `border-radius: 50%`

Reply:
Semi fixed. I wasn't able to move it out of `.sm` and `.md`